lm_eval/tasks/jsquad.py

```diff
@@ -188,12 +163,40 @@ def aggregation(self):
         }
     
     def higher_is_better(self):
-        """
-        :returns: {str: bool}
-            A dictionary where keys are the names of submetrics and values are
-            whether a higher value of the submetric is better
-        """
         return {
             "exact_match": True,  # Exact match (the normalized answer exactly match the gold answer)
             "f1": True,  # The F-score of predicted tokens versus the gold answer
         }
+
+
+class JSQuADForFintan(JSQuAD):
```

Comment:
Could we change the classes which simply use different prompt template to the format `{task_name}With{Source}Prompt`? I think it will be easy for others to understand the essence of the class.

For this case, it will be named to `JSQuADWithFintanPrompt`. A bit longer but I prefer the clarity, how do you think?

Reply:
I don't have particular preference about this point so I changed as you suggested. 
https://github.com/Stability-AI/lm-evaluation-harness/pull/1/commits/9688c25fd22b67aa0e95b1a0d2ce11ff47b7bc91